pkg/agent/openflow/pipeline_windows.go

```diff
@@ -23,6 +23,13 @@ import (
 	binding "antrea.io/antrea/pkg/ovs/openflow"
 )
 
+// matchUplinkInPort matches dstIP field to prevent unintended forwarding when promiscuous mode is enabled on Windows.
+func (f *featurePodConnectivity) matchUplinkInPort(flowBuilder binding.FlowBuilder) binding.FlowBuilder {
+	return flowBuilder.
+		MatchProtocol(binding.ProtocolIP).
```

Comment:
ditto, also adding "flowBuilder.MatchInPort(f.uplinkPort)" inside this function?

Reply:
Got it, code updated.